packages/core/admin/admin/tsconfig.build.json

```diff
@@ -1,9 +1,23 @@
 {
-  "extends": "./admin/tsconfig.json",
-  "include": ["./admin/src/*", "./admin/custom.d.ts", "./shared/*", "./admin/module.d.ts"],
-  "exclude": ["tests/*", "**/*.test.*"],
+  "extends": "tsconfig/client.json",
   "compilerOptions": {
-    "rootDir": ".",
+    "rootDir": "../",
+    "baseUrl": ".",
     "outDir": "./dist"
-  }
+  },
+  "include": [
+    "./src",
+    "./custom.d.ts",
```

Comment:
just while you're at it, there's a "duplicate identifier promoteEE" in that file, could you fix it too?

Reply:
I can 😎